Form-Controls/index.html

```diff
@@ -10,18 +10,100 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1 class="design-h">::<span class="h1-style">::</span>::<span class="h1-style">::</span>
+    </h1>
   </header>
+
+  <!-- form and image section -->
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div class="forme-wrapper">
+  <form action="">
+    <h1>ORDER NOW</h1>
+    <!-- shoping logo -->
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="orangered"
+      viewBox="0 0 24 24"
+      stroke-width="1.5"
+      stroke="white"
+      class="w-6 h-6"
+      width="40px"
+      height="40px"
+    >
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        d="M13.5 21v-7.5a.75.75 0 01.75-.75h3a.75.75 0 01.75.75V21m-4.5 0H2.36m11.14 0H18m0 0h3.64m-1.39 0V9.349m-16.5 11.65V9.35m0 0a3.001 3.001 0 003.75-.615A2.993 2.993 0 009.75 9.75c.896 0 1.7-.393 2.25-1.016a2.993 2.993 0 002.25 1.016c.896 0 1.7-.393 2.25-1.016a3.001 3.001 0 003.75.614m-16.5 0a3.004 3.004 0 01-.621-4.72L4.318 3.44A1.5 1.5 0 015.378 3h13.243a1.5 1.5 0 011.06.44l1.19 1.189a3 3 0 01-.621 4.72m-13.5 8.65h3.75a.75.75 0 00.75-.75V13.5a.75.75 0 00-.75-.75H6.75a.75.75 0 00-.75.75v3.75c0 .415.336.75.75.75z"
+      />
+    </svg>
+    <!-- <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAAAXNSR0IArs4c6QAAAzxJREFUaEPtmVmoTlEUx383XoyZ86BEZkIyRUIevCgeSEKJ6EpExuRNZJZSpihDEkUpr0KU4cU8ZogyRCJDXgz965xatr3POfd+537HrbNevr611157/dfa/7X3t78aGrnUNPL4KQEUXcG6VKAdsB4YB/RroMAfAOeBNcDXLGtkBdACuAoMyOI0B5vLwETge5qvrAD2A/PTnOU8vhdYmOYzC4BhwDWoOuF/A4OB20kg0gBo/CYw0DiRwyHAT6PbBiw33zdH+9iuvRVYYRRbgNXmexPgLtDH6G4AIwCB8UoagFpgjzNzOCDHVi4AY41iOnDSsZkBHDc6kXWCYzMa0P63sgA4UB8AHYAnQBsz+TAwx+NMZGtm9D2Ap45dL+CR0X12fMdDR4FZxu4T0A3Q5z+SVIGDwFwz4wvQHfjgeFHJ1f5iERh1LZ98A5qbAYFSkqwocc+AVka5D9BuyAxAxL3uWC8Gdnt8zAaOGL220/gAgEvAGDOmbXXCY7sU2Gn0QUL7KhAirjqCj0y7gCVmse0OWW18CkrBxRKyzUxoH4BFnkz7iBsHcQUYZYKa6ZDVAnCrdTE62X0Fy0RoH4A3QGfj8RAwL7AlpBY3WppxccKS1U7tC9w3io9A+wTfLqFfAl2tvQ+A21FE3OeBRboAr5yxjh6ixyadgHeOvZLl6mKTnsBjY69ktU4D8BDobYwmAecCANSl1K2sTAbOBuynAqecMW2rYwH7KcAZM3bHOVS914MdwDIz6RYwEvjhLKLs61Ru6+iVsUEee7VPnbTq6VbeA/0BfVrRuaILpL0FbADWpVVAe+yF40zIV0YncNOItOrN2hI+UaCrojuU7JWATYA44JPX0WVRAasDDQV01bC3XyVQ4N+mAdC4kK4NLFaUWplXXH9J0kkcvEAVhMAbawiA9L8KCjS0bAmg6IKUFYgrkET8huRObhUoAdSTTGUFKuVApVuv8Ar8NwDquYUTH8aydK/cKlAC8GSgrIDvmp33bbQksY981fw9UFagLhWQrV4X9C6Th9zL8PdU0nrB+UllnQZsBPRUXono5Vo/yE+nOAmtlzg/7Q+OSgKvytwSQFXSnLBIWYGyAhVm4A9aA6kxUHhNmgAAAABJRU5ErkJggg==" class="logo"/> -->
+
+<div class="name-wrapper">
+    <div class="input-box">
+<input type="text" placeholder="Name" required>
+<i class='bx bxs-user'></i>
+    </div>
+     
+<div class="input-box">
+<input type="text" placeholder="lastname" required>
+    </div>
+</div>
+
+   <div>
+          <label for="color">Color</label>
+          <br />
```

Comment:
Instead of this br tag, what could you have used?

Why do we avoid using br tags?

Reply:
we use display block, and also other way through (flex box), we avoid using br tag because if we want a responsive page then we must avoid using br tag.